transports/quic/src/config.rs

```diff
@@ -132,6 +135,7 @@ impl From<Config> for QuinnConfig {
             keep_alive_interval,
             max_connection_data,
             max_stream_data,
+            #[expect(deprecated)]
```

Comment:
Same here. Move it to apply to the whole `impl From<Config> for QuinnConfig` block.